src/Google.Api.Gax.Rest/UserAgentHelper.cs

```diff
@@ -6,8 +6,7 @@
  */
 
```

Comment:
Have updated docs to include "and related headers" which I think is okay :) (And it means we don't need to make breaking changes outside GAX)